### DISCUSSION TITLE
Would anyone happen to have any other method, My school has disabled the inspect command ( Ouch )

### DISCUSSION THREAD

**Discussion by ScopfyScythe** [3kh0/ext-remover#324] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Would anyone happen to have any other method, My school has disabled the inspect command ( Ouch )

**Comment by 3kh0** [comment, marked as answer]
Check it again, one was added

**Comment by Aka-but-nice** [comment]
For some reason that doesn't work, or at least for me. the blank page loaded up I put javascript and then the let shim thing in the browser and it ran the command so I put the extension id in, and I saw that the extension would disable or reload very fast but chrome would fix the issue and bring it back but it would keep disabling over and over, but when I went to a blocked site like 1v1.lol it would suddenly stop disabling and work normally.

**Reply by Aka-but-nice** [reply]
If that doesn't also work for you, try this. For this method make a bookmark with chrome://signOut the o has to be capitalized, then make a bookmark with chrome://restart in it, once you have those, press the chrome://signOut bookmark and press the blue button, then click the chrome://restart bookmark and let it load and you're good. Now, this is not mine at all credit to whoever found this method!
